src/multiplex-codec.js

```diff
@@ -0,0 +1,3 @@
+'use strict'
+
+module.exports = '/mplex/6.7.0'
```

Comment:
Should be `multiplex` https://github.com/libp2p/specs/blob/master/7-properties.md#757-protocol-multicodecs, please submit that patch to go-libp2p as well